dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp

```diff
@@ -1,20 +1,22 @@
 <%@ page import="com.liferay.portal.language.LanguageUtil" %>
-<form dojoAttachPoint="search_form" onsubmit="return false;" id="searchForm" class="related-content-form">
-    <div dojoType="dijit.layout.BorderContainer" design="sidebar" gutters="false" liveSplitters="true" dojoAttachPoint="borderContainer" class="related-content-container">
+<form dojoAttachPoint="search_form" onsubmit="return false;" id="searchForm" >
+<div style="display:flex;flex-direction: row;">
+	
 
         <!-- START Left Column -->
-        <div dojoType="dijit.layout.ContentPane" splitter="false" region="leading" class="portlet-sidebar-wrapper">
+        <div class="portlet-sidebar-wrapper" style="width:200px;min-height:700px;">

```

Comment:
The style changes should be inside the `portlet-sidebar-wrapper` class

Reply:
done